docs/lakehouse-iceberg-catalog.md

```diff
@@ -268,13 +268,21 @@ The Gravitino server doesn't allow passing the following reserved fields.
 |---------------------------------|---------------------------------------------------------|
 | `comment`                       | The table comment.                                      |
 | `creator`                       | The table creator.                                      |
-| `location`                      | Iceberg location for table storage.                     |
 | `current-snapshot-id`           | The snapshot represents the current state of the table. |
 | `cherry-pick-snapshot-id`       | Selecting a specific snapshot in a merge operation.     |
 | `sort-order`                    | Selecting a specific snapshot in a merge operation.     |
 | `identifier-fields`             | The identifier fields for defining the table.           |
 | `write.distribution-mode`       | Defines distribution of write data                      |
 
+Gravitino server doesn't allow to change such properties:
+
+| Configuration item | Description                                  |
```

Comment:
I think the format of this table should be consistent with other documents. For example:
https://github.com/apache/gravitino/blob/397b338068a120f3bb3e3256a4a9b4d3a75f5ca2/docs/apache-hive-catalog.md?plain=1#L136-L144

Reply:
ok